### PR TITLE
Export GitHub token

### PR DESCRIPTION
**Pull request by amitkrout**
**What type of PR is this?**
/kind enhancement

**What does this PR do / why we need it**:
Github token is needed for kam day 1 operation
**Have you updated the necessary documentation?**

* [ ] Documentation update is required by this PR.
* [ ] Documentation has been updated.

**Which issue(s) this PR fixes**:

Fixes part of https://issues.redhat.com/browse/GITOPS-333

**How to test changes / Special notes to the reviewer**:
Token should be exported in the test container

### CODE REVIEW

File: scripts/openshiftci-presubmit-all-tests.sh [hunk 1]
```diff
@@ -39,4 +43,5 @@ else
     exit 1
 fi
 
+# kam version check
```
Comment (bigkevmcd) [inline scripts/openshiftci-presubmit-all-tests.sh:46]: What's this comment indicating?

File: scripts/openshiftci-presubmit-all-tests.sh [hunk 2]
```diff
@@ -2,9 +2,13 @@
 
 # fail if some commands fails
 set -e
+
+# Do not show token in CI log
+set +x
+export GITHUB_TOKEN=`cat $KAM_GITHUB_TOKEN`
```
Comment (bigkevmcd) [inline scripts/openshiftci-presubmit-all-tests.sh:8]: This implies that `KAM_GITHUB_TOKEN` is probably badly named, maybe something like `KAM_GITHUB_TOKEN_FILE` ?

Comment (amitkrout) [reply scripts/openshiftci-presubmit-all-tests.sh:8]: Pr https://github.com/openshift/release/pull/13832 will update the new name. PTAL and apply `/lgtm` label.

File: scripts/openshiftci-presubmit-all-tests.sh (hunk 1 above)
Comment (amitkrout) [reply scripts/openshiftci-presubmit-all-tests.sh:46]: I just add a comment line for `kam version` command

Comment (bigkevmcd) [reply scripts/openshiftci-presubmit-all-tests.sh:46]: But it's not checking the version?

Comment (amitkrout) [reply scripts/openshiftci-presubmit-all-tests.sh:46]: Its checking the existence of the correct kam binary in the PATH by running the version command. I can re phrase to a better comment for example ```Checking kam binary in the PATH through version command.``` ?

Comment (bigkevmcd) [reply scripts/openshiftci-presubmit-all-tests.sh:46]: How about something like this?

```bash
# assert that kam is on the path
```

**Review by bigkevmcd — APPROVED** [review]

File: scripts/openshiftci-presubmit-all-tests.sh [hunk 3]
```diff
@@ -2,9 +2,13 @@
 
 # fail if some commands fails
 set -e
+
+# Do not show token in CI log
+set +x
+export GITHUB_TOKEN=`cat $KAM_GITHUB_TOKEN_FILE`
```
Comment (wtam2018) [inline scripts/openshiftci-presubmit-all-tests.sh:8]: Where is `KAM_GITHUB_TOKEN_FILE`defined?

Comment (amitkrout) [reply scripts/openshiftci-presubmit-all-tests.sh:8]: https://github.com/openshift/release/blob/master/ci-operator/config/redhat-developer/kam/redhat-developer-kam-master__v4.5.yaml#L42

**Pull request merged**